### DISCUSSION TITLE
Version TestFlight/APK ou version App Store/Play Store ?

### DISCUSSION THREAD

**Discussion by TheTiger72** [PapillonApp/papillon-v5#104] [Q&A] [answered]
J'ai remarqué que par moment les mises à jours les plus récentes apparaissent d'abord sur les versions comme sur TestFlight mais dès fois les versions les plus récentes apparaissent d'abord en release sur les stores.

Donc je ne sais pas sur quelle version resté pour avoir les dernières mises à jour de fonctionnalités et de correction de bug.

**Comment by sivelswhy** [comment]
Normalement, le deploy est automatique sur les deux plateformes, le temps que tu attend, c'est le temps que le store approuve la version

**Reply by TheTiger72** [reply]
Ah d'accord, je comprends mieux, merci

**Comment by lucas-luchack** [comment, marked as answer]
Les versions sont d'abord publiés en **BETA** afin qu'elles puissent être "_debuggées_" avant une publication sur les stores *officiels*.

Il arrive parfois que certaines versions soit disponible directement sur les stores officiels sans passer par la beta. Cependant, le choix du canal de diffusion de votre part n'a pas d'importance. Que ce soit sur Testflight ou sur le Play Store le canal BETA reçoit **toute les versions**.

**Reply by TheTiger72** [reply]
D'accord merci !
